test/census_test.go

```diff
@@ -213,18 +215,18 @@ func TestCensus(t *testing.T) {
 
 	// getRoot
 	resp = doRequest("getRoot", nil)
-	root = resp.Root
+	root = testutil.Hex2byte(t, resp.Root)
 	if len(root) < 1 {
 		t.Fatalf("got invalid root")
 	}
 
 	// getRoot from published census and check censusID=root
-	req.CensusID = root
+	req.CensusID = hex.EncodeToString(root)
```

Comment:
Is not better to use `fmt.Sprintf("%x", root)` instead? We use fmt in general for hex encoding.

Reply:
I intend to move CensusID and the other fields in part 2, so I don't think it matters :)